popup.css

```diff
@@ -0,0 +1,319 @@
+#popup-modal {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: rgba(235, 236, 240, 0.941);
+  width: 100%;
+  height: 100vh;
+  z-index: 10;
+  padding: 0;
+  overflow: auto;
+}
+
+.popup {
+  display: flex;
+  align-items: start;
+  justify-content: start;
+  gap: 8px;
+  position: relative;
+  height: auto;
+  width: auto;
+  margin: 15px 0 auto 0;
+  padding: 4%;
+  border-radius: 6px;
+  flex-direction: column;
+  border: 1px solid #ebecf0;
+  background-color: #fff;
+  z-index: 99;
```

Comment:
- [ ] Currently, The close button on your pop-up is not visible, or it's not correctly positioned as users can see the close button when they open the popup. :zap:  Please you need to look into this as it's part of the requirement, and also the users need a way to interact with the popup once it's open. :smile: :wave: 
- Vdieo for reference:

https://user-images.githubusercontent.com/107632780/233812017-450ad244-f4c3-495e-9fca-bf266ea4597c.mp4